features/dd-sdk-android-session-replay/src/main/res/values/ids.xml

```diff
@@ -6,4 +6,6 @@
 
 <resources>
     <item name="datadog_hidden" type="id"/>
+    <item name="datadog_image_privacy" type="id"/>
```

Comment:
can be these used later when we're going to introduce the namespace for xml templates ?

Reply:
I'm not sure that the namespace will be practical. It would seem to be using custom attributes, however custom attributes can [only be read](https://developer.android.com/develop/ui/views/layout/custom-views/create-view) when a view is created and not afterwards. Let's discuss